src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -352,7 +358,13 @@ static ucs_status_t
 uct_rdamcm_cm_ep_set_qp_num(struct rdma_conn_param *conn_param,
                             uct_rdmacm_cm_ep_t *cep)
 {
-    return uct_rdamcm_cm_ep_create_qpn(cep, &conn_param->qp_num);
+    ucs_status_t status;
+
+    status = uct_rdamcm_cm_ep_create_qpn(cep, &conn_param->qp_num);
+    if (status == UCS_OK) {
+        cep->flags |= UCT_RDMACM_CM_EP_QPN_CREATED;
```

Comment:
sorry if I was not clear, pls move back `cep->flags |= UCT_RDMACM_CM_EP_QPN_CREATED;` to `cep->flags |= UCT_RDMACM_CM_EP_QPN_CREATED;`, but `conn_param->qp_num` has to be initialized here

Reply:
@evgeny-leksikov Fixed